packages/grafana-schema/src/schema/mudball.cue

```diff
@@ -128,7 +128,11 @@ GraphThresholdsStyleConfig: {
 LegendPlacement: "bottom" | "right" @cuetsy(kind="type")
 
 // TODO docs
-LegendDisplayMode: "list" | "table" | "hidden" @cuetsy(kind="enum")
+LegendDisplayMode: "list" | "table" @cuetsy(kind="enum")
+
+// TODO docs
+
+LegendVisibility: "visible" | "hidden" @cuetsy(kind="enum")
```

Comment:
Not sure I understand why we need this new enum. Can't we rely on the `LegendDisplayMode`? Think it would be better given that this change implicates a need for migration.

Reply:
Damn, I get it - it's probably to make sure that the given legend type(list/table) stays the same when one toggles it back on.